<div class="landing-modal-panel">
    <div class="row">
        <div class="col s12 ">
            <h1 class='landing-h1'>Oncoscape</h1>
            <h2 class='landing-h2'>Acclerate your research</h2>
        </div>
    </div>
    <div class="row">
        <div class=' landing-marque '>
            <div class='landing-marque-image '>
                <img class="landing-image" src="/assets/landing/img-1.jpg" alt="knot">
            </div>

            <div class='landing-marque-image '>
                <img class="landing-image" src="/assets/landing/img-4.jpg" alt="M+P">
            </div>

            <div class='landing-marque-image '>
                <img class="landing-image" src="/assets/landing/img-chromo.jpg" alt="chromosome">
            </div>

            <div class='landing-marque-image hide-on-med-and-down '>
                <img class="landing-image " src="/assets/landing/img-genome.jpg" alt="genome">
            </div>

            <div class='landing-marque-image hide-on-med-and-down '>
                <img class="landing-image " src="/assets/landing/img-scatter.jpg" alt="clustering">
            </div>
        </div>
    </div>
    <div class="row">
        <div class="col s12">
            <a href="#" (click)="getStartedClick()" class='landing-link '>Get Started</a>
        </div>
    </div>

    <section class='row '>
        <h2 class='landing-copy-h2 '>Introducing Oncoscape 3.0</h2>
        <p class='landing-copy '>
            The rapid growth of cloud computing, public data sets and analytic methods presents both opportunities and challenges. STTR
            develops software products and provides professional services that enable researchers and clinicians to extract
            maximal biological mean from complex genomic data. To accelerate research the STTR developed Oncoscape, an omics-focused
            data visualization platform. The solution enables researchers to analyze and augment their clinical and molecular
            data with industry standard analytic libraries and NCI sponsored datasets respectively. To learn more or request
            a custom solution, contact us at contact@oncoscape.org
        </p>
    </section>
    <section class='row '>
        <h2 class='landing-copy-h2 '>What's New</h2>
        <p class='landing-copy'>
            Random Text... Our team assists researchers and clinicians extract maximal biological meaning from human genomic data through
            the development of custom software solutions. Our mission is to accelerate your science. Today, researchers wanting
            to augment their own datasets or analytic methods face many challenges including: identifying, downloading and
            storing data from multiple public repositories; developing scripts to integrate and interrogate these resources;
            creating novel data visualization to extract meaning. To help mitigate these challenges, the STTR has built a
            cloud-based platform called Oncoscape. Our platform allows researchers to easily upload mutation, expression,
            copy number, structural variation, clinical, and time series data; augment their data with a growing list of
            reference datasets including TCGA, Reactome, MSigDB, HGNC and BioMart; interrogate their data using industry
            standard libraries such as scikit learn, lifelines and bioC; Visualize data using any one of our XX tools. To
            learn more or request custom analysis contact oncoscape@fredhutch.org
        </p>
    </section>
    <section class='row'>
        <h2 class='landing-copy-h2'>Analysis</h2>
        <dl class='landing-copy'>
            <dt>Matrix Decomposition</dt>
            <dd>Visualize the inherent spatial structure of high dimensional datasets in 3D. With 13 fully customizable linear
                dimension reduction algorithm, select from 5 variants of PCA, Factor Analysis, LDA, SVD and more.</dd>

            <dt>Manifold Learning</dt>
            <dd>Like Matrix Decomposition, Manifold Learning seeks to unveil the spatial structure of high dimensional datasets.
                Unlike Matrix Decomposition the dimension reduction process is non-linear. Available methods include T-SNE,
                MDS, IsoMap, Local Linear and Spectral Embedding.</dd>

            <dt>Heat map</dt>
            <dd>Sort, group and visualize your data using 7 agglomerative clustering algorithms in conjunction with 20 distance
                metrics. Leverage dendrogram to sub select populations of samples or markers for further investigation.</dd>

            <dt>Genome</dt>
            <dd>Identify chromosomal regions of interest by projecting your data onto a visual representation of the genome.
                Overlay CNV, Gistic or expression data along with Topically Aligned Domains and Cytobands. Explore thousands
                of curated gene sets or create your own.</dd>

            <dt>Chromosome</dt>
            <dd>Compare datasets at the chromosomal level with circos and linear visualizations. Overlay pathways or topically
                aligned domains.
            </dd>

            <dt>Pathways</dt>
            <dd>Overlay your data on over 5,900 pathways curated from Kegg, Panther, Reactome, SMPDB, WikiPathways and Pathway
                Commons.
            </dd>


            <dt>Survival + Hazard</dt>
            <dd>Estimate and compare survival and hazard rates for patient populations using Kaplin Kaplan-Meier, Nelson-Aalen,
                Aalen, Weibull and Cox models. </dd>

            <dt>Timelines</dt>
            <dd>Visualize patterns in time series data with an interactive swimmer plot. Sort, group and highlight patients using
                clinical attributes.</dd>

            <dt>Box + Whisker</dt>
            <dd>Gain insights into the distribution of your data by visualizing the minimum, first quartile, median, 3rd quartile
                and maximum values.</dd>

            <dt>Fore Directed Graph</dt>
            <dd>Visualize graph data in three-dimensions using a class of algorithms called force directed graphs. These algorithms
                use the number and strength of connections to position nodes.</dd>

            <dt>Spreadsheet</dt>
            <dd>Sort and export your data, cohorts and genes of interest in tabular format.</dd>

            <dt>Clinical Dashboard</dt>
            <dd>Preview </dd>

            <dt>Edges</dt>
            <dd>Connect multiple graphs with edges. Link tumor/normal samples or visualize mutations.</dd>

            <dt>Cohort Builder</dt>
            <dd>Preform ad-hoc queries for patients with defined characteristics. Isolate patients with specific mutations or
                clinical attributes.
            </dd>

            <dt>Gene Sets</dt>
            <dd>Select from thousands of curated gene sets from MSigDB or create your own. </dd>

            <dt>Data Upload</dt>
            <dd>Visualize your RNA, CVN, Mutation, Event, Patient, Sample data in Oncoscape. To get started download an excel
                template and follow the instructions at http://needurl.com/</dd>

            <dt>Reproducibility</dt>
            <dd>All analysis in Oncoscape is fully reproducible. All data manipulation and augmentation is done using best in
                class / open source libraries and data stores with citations. Pause, rewind and interrogate every variable
                and result.</dd>

        </dl>
    </section>
</div>